### PR TITLE
Chore: New util smallerVersion

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

We also need smallerVersion in the hardware-wallet-cli.

# Changes

* Add new util smallerVersion in utils package.

# Tests

* Test the new util.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker please review

File: packages/utils/src/utils/version.utils.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * @param {Object} params
+ * @param {string} params.minVersion Ex: "1.0.0"
+ * @param {string} params.currentVersion Ex: "2.0.0"
+ * @returns boolean
+ */
+export const smallerVersion = ({
+  minVersion,
+  currentVersion,
+}: {
+  minVersion: string;
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply packages/utils/src/utils/version.utils.ts:18]: this gist was posted in the issue you linked e.g. https://gist.github.com/iwill/a83038623ba4fef6abb9efca87ae9ccb

[the remark this answers is not in the document]
Comment (lmuntaner) [reply packages/utils/src/utils/version.utils.ts:18]: Good point.

Let me fix that.

[the remark this answers is not in the document]
Comment (lmuntaner) [reply packages/utils/src/utils/version.utils.ts:32]: done

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

Asked my self if naming the function `smallerSemanticVersion` or `smallerSemVer` would make its goal more ovious? I'm fine with `smallerVersion`, just mentionning it in case it's an interesting input.

**Comment by lmuntaner** [comment]
> LGTM, thanks
> 
> Asked my self if naming the function `smallerSemanticVersion` or `smallerSemVer` would make its goal more ovious? I'm fine with `smallerVersion`, just mentionning it in case it's an interesting input.

I don't have a strong opinion, but if you don't either, let's leave it as is for now.

**Pull request merged**